### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by LilJohny**

### CODE REVIEW

File: src/components/ArticleData.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import AvatarImage from "./AvatarImage";
+import '../css/ArticleData.css'
+
+export default class ArticleData extends React.Component {
+    render() {
+        return (
+            <div className="ArticleData">
+                <div className={"flexbox"}>
```
Comment (pavlo-berezin) [inline src/components/ArticleData.js:9]: On top of this line you use this syntax `className="ArticleData"` and here `className={"flexbox"}`. Please keep consistent.

Comment (LilJohny) [reply src/components/ArticleData.js:9]: Thanks, I'll fix it.

File: src/components/BackToTopButton.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+function ScrollTop(props) {
+    const {children} = props;
+    const classes = useStyles();
+    const trigger = useScrollTrigger({
+        disableHysteresis: true,
+        threshold: 100
+    });
+
+    const handleClick = event => {
+        const anchor = (event.target.ownerDocument || document).querySelector(
```
Comment (pavlo-berezin) [inline src/components/BackToTopButton.js:25]: Consider using refs instead of selecting.

File: src/components/IconsList.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    return (
+        <div className={classes.root}>
+            <List component="nav" aria-label="main mailbox folders">
+                {props.list.map((text, index) => (
+                    <ListItem button>
+                        <ListItemIcon>
+
+                            <DraftsIcon/>
+                    </ListItemIcon>
```
Comment (pavlo-berezin) [inline src/components/IconsList.js:32]: Please fix indentation.

File: src/pages/profile.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            gridColumn: "1/span 7"
+        };
+        return (
+            <React.Fragment>
+                <NavBar/>
+                <Container className={"profile"}>
+                    <Paper>
+                        <article>
+                            <div className={"flexbox"}>
+                            <span><AvatarImage large={true} variant={"rounded"} className={"ProfilePicture"}
```
Comment (pavlo-berezin) [inline src/pages/profile.js:25]: For containing several elements use block-elements instead of `span`'s.

File: src/css/Article.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+article > ul > li > p, article > p, .link {
+    --x-height-multiplier: 0.375;
+    --baseline-multiplier: 0.17;
+    font-family: Georgia, Cambria, "Times New Roman", Times, serif;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 21px;
+    line-height: 1.58;
+    letter-spacing: -0.003em;
```
Comment (pavlo-berezin) [inline src/css/Article.css:63]: Does it make any difference?

**Comment by pavlo-berezin** [comment]
Nice work.

Please use some kind of formatter, because your formatting is not consistent.

File: src/pages/profile.js (hunk 4 above)
Comment (LilJohny) [reply src/pages/profile.js]: Fixed

**Pull request merged**